src/javascript/_common/base/livechat.js

```diff
@@ -2,13 +2,9 @@ const BinarySocket  = require('./socket_base');
 const ClientBase = require('./client_base');
 
 const LiveChat = (() => {
-    const initial_session_variables = { loginid: '', landing_company_shortcode: '', currency: '', residence: '' };
+    const initial_session_variables = { loginid: '', landing_company_shortcode: '', currency: '', residence: '', email: '' };
```

Comment:
```suggestion
    const initial_session_values = { loginid: '', landing_company_shortcode: '', currency: '', residence: '', email: '' };
```

Reply:
I think variable is fine, since it's a key pair, not just a value